src/App.js

```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md)for this project, " `Use class based components (All components should be class-based).` ", please kindly change your `App.js` component which a currently a functional component to a class-based component.

Reply:
Hi @Mihndim2020 

Shall I delete the files which are in the Public folder or there is something else?